src/pybind/mgr/cephadm/module.py

```diff
@@ -1301,7 +1301,7 @@ def describe_service(self, service_type=None, service_name=None,
                     if dd.daemon_type == 'osd':
                         """
                         The osd count can't be determined by the Placement spec.
-                        It's rather pointless to show a actual/expected representation 
+                        It's rather pointless to show a actual/expected representation
```

Comment:
```suggestion
                        Showing an actual/expected representation cannot be determined
```

Reply:
minor nit

Reply:
For some clarification, my text editor seems to delete white space at the end of lines on its own sometimes and this line "change" doesn't actually have anything to do with the purpose of this PR. That said, I'll still change the wording like you recommended as I think your wording sounds a little better and I see no reason not to.